src/controllers/userController.js

```diff
@@ -0,0 +1,76 @@
+const userService = require('../services/userService');
+
+const get = async (req, res) => {
+  res.send(await userService.getAllUsers());
+};
+
+const getOne = async (req, res) => {
+  const { id } = req.params;
+
+  const user = await userService.getUserById(id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(user);
+};
+
+const update = async (req, res) => {
+  const { id } = req.params;
+  const { name } = req.body;
+  const user = await userService.updateUser({ id, name });
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  if (typeof name !== 'string') {
+    res.sendStatus(422);
+
+    return;
+  }
+
+  res.send(user);
+};
+
+const create = async (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
```

Comment:
You should validate the 'name' before creating the user. If the name is not a string or is an empty string, you should return a 400 status code (Bad Request). This check should be done before calling the 'createUser' method.